pkg/platform/openstack/networking.go

```diff
@@ -366,3 +366,17 @@ func ensureOpenStackSgRule(client *gophercloud.ServiceClient, sgId, remotePrefix
 	}
 	return nil
 }
+
+// Returns list of OpenStack ingress security group rules on sgId SG.
+func listOpenStackSgRules(client *gophercloud.ServiceClient) ([]rules.SecGroupRule, error) {
+	page, err := rules.List(client, rules.ListOpts{Direction: "ingress"}).AllPages()
```

Comment:
what about the tag? and if this is only for master/workers, why not restrict the search to those SGs?

Reply:
Nice idea!

Reply:
Oh crap, seems like we don't tag pod SG yet…

Reply:
Nevermind.